src/gui/editors/PianoRoll.cpp

```diff
@@ -712,6 +712,89 @@ void PianoRoll::glueNotes()
 	}
 }
 
+void PianoRoll::fitNoteLengths(bool shrink, bool grow)
+{
+	if (hasValidPattern())
+	{
+		m_pattern->addJournalCheckPoint();
+
+		NoteVector notes = getSelectedNotes();
+		if (notes.empty())
+		{
+			notes = m_pattern->notes();
+		}
+
+		// Sort notes by position
+		std::sort(notes.begin(), notes.end(), [](const Note *n1, const Note *n2){ return n1->pos() < n2->pos(); });
```

Comment:
```suggestion
		std::sort(
			notes.begin(),
			notes.end(),
			[](const Note *n1, const Note *n2){ return n1->pos() < n2->pos(); }
		);
```

Just breaking the line to shorter ones.